apis/Google.Cloud.ErrorReporting.V1Beta1/docs/index.md

```diff
@@ -40,6 +40,12 @@ optionally specifying a service endpoint and settings.
 
 [!code-cs[](obj/snippets/Google.Cloud.ErrorReporting.V1Beta1.ReportErrorsServiceClient.txt#ReportErrorEvent)]
 
+## Automatic Error Reporting for ASP.NET
```

Comment:
I'd suggest to move this section before the 'Report an error' section.

Also, I think you should also mention the Diagnostics ASP.NET package at the very top in the intro paragraph.